### ISSUE TITLE
Character ported out of raid fight when DC'ing

### ISSUE THREAD

**Issue by dennots** [chromiecraft/chromiecraft#3461]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59
- [X] 60

### Current Behaviour

We had a situation in Molten Core where a raid member DC'd momentarily before we pulled Golemagg, we proceeded with the pull not noticing this quick disconnect, when he reconnected a minute later he appeared outside the raid entrance in BRD instead of inside the boss room were he initially got DC'd, unable to walk in getting a "Fight in progress" message.

### Expected Blizzlike Behaviour

Usually you reappear where you logged off inside a dungeon or raid if there is space in the instance. You would only get ported outside if the instance was filled while you were offline.

### Steps to reproduce the problem

1. Get inside a raid instance
2. Log off before Boss gets engaged.
3. Log back in after Boss in engaged.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/283ce18bb7b961264eae983a401aaacb89725273

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by Dejanos** [comment]
Hey dennots, do you perhaps have any source of your claim that is Blizzlike that you get to continue with the boss fight once you get a disconnect while boss is engaged?

**Comment by Nyeriah** [comment]
Yes, from retail experience, that's what happens. Disconnecting during a fight doesn't kick you out of the instance.

**Comment by Stump876** [comment]
Agreed, It should just make your character inactive until you time out, which takes about 2 minutes, but it would never kick you. The character would have the little lighning bolt through their portait. On log in you should end up in the same spot from where you dc'd. Triage group discussed this for a bit a while ago, but it seems nobody updated this. Maybe we could have someone do a check on retail. 

This is what I could find dong a quick look. Both times seems like the character was still present in the raid at very inconvineint times.
https://gamefaqs.gamespot.com/boards/534914-world-of-warcraft/54085065?page=2
![image](https://user-images.githubusercontent.com/68164076/188333780-c23decb8-8996-4176-9756-f3d93c56359a.png)
![image](https://user-images.githubusercontent.com/68164076/188333849-f5949fa3-ebcd-4d79-a525-9cb29b05b210.png)
